rspack.config.js

```diff
@@ -37,33 +37,38 @@ const OPTIMIZATIONS = {
 /**
  * @param {boolean} lit
  */
-const postcssLoader = (lit = false) => ({
-  loader: "postcss-loader",
-  options: {
-    postcssOptions: {
-      plugins: [
-        [
-          "@csstools/postcss-global-data",
-          {
-            files: [
-              "./components/media/index.css",
-              "./components/layout/global.css",
-            ],
-          },
-        ],
-        ["postcss-custom-media"],
-        [
-          "postcss-preset-env",
-          {
-            stage: 2,
-            minimumVendorImplementations: 2,
-          },
+const postcssLoaders = (lit = false) => [
+  {
+    loader: "./build/loaders/fix-light-dark.js",
+  },
```

Comment:
Nit: Would be good to have a comment explaining (beyond git log) why this is necessary.